components/angular-tomitribe-tags/src/tags.configurer.ts

```diff
@@ -1,3 +1,12 @@
 export class TagsConfigurer {
-  tagsEndpoint = 'api/label/available';
+    tagsEndpoint = 'api/label/available';
+
+    pattern: string = "^[A-Za-z0-9 ._-]*$";
```

Comment:
Can you combine tagging configuration into a block so it will take a form like (just combined into basic structure, not real code )):
```
tagging = {
    default: {
        pattern: string = "^[A-Za-z0-9 ._-]*$",
        maxLength: number = 64,
        errorMessage: string = "Inputs must be alphanumeric, dot, space, scores and cannot exceed " + this.maxLength + " chars. Please remove invalid entry(s).",
        isValid(input: any): boolean {
            var regex = new RegExp(this.pattern);
            return (input && input.length <= this.maxLength && regex.test(input));
        }
    }
}
```
and add attribute as a property of tagging validation, so we can add different validations like positive number or some alphanumeric string without spaces in future.